_build/install-kscs

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -ef
+# Downloads and installs different versions of ksc for testing purposes
+
+# Create directory for all ksc versions
+mkdir -p ksc
+cd ksc
+
+# Download normal releases
+#
+# 0.6 is also possible to download, but we're intentionally skipping it
+# due to the fact that it's unable to produce JSON output
+for VER in 0.7 0.8; do
+	echo "Installing $VER"
+	rm -rf "$VER" "kaitai-struct-compiler-$VER"
+	wget "https://dl.bintray.com/kaitai-io/universal/$VER/kaitai-struct-compiler-$VER.zip"
+	unzip "kaitai-struct-compiler-$VER.zip"
+	mv "kaitai-struct-compiler-$VER" "$VER"
+done
+
+# Download latest unstable
+wget https://bintray.com/kaitai-io/universal_unstable/kaitai-struct-compiler/_latestVersion
```

Comment:
shouldn't `apt` be used for that?

Reply:
No, the whole reason is that we download and use multiple versions simultaneously. There's little point in using apt repo for that.